packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -278,6 +278,11 @@ module.exports = {
               },
             ],
           },
+          // The GraphQL loader preprocesses GraphQL queries in .graphql and .gql files.
+          {
+            test: /\.(graphql|gql)$/,
```

Comment:
I prefer us to be opinionated here and pick `.graphql` as the extension. It's not obvious what `.gql` is, it has to be repeated everywhere, syntax highlighters have to support both, etc.

Reply:
Not sure if I agree there as both extensions have been adopted by the userland at large from my observations. For example the [Apollo references both](https://www.apollographql.com/docs/react/recipes/webpack.html) even while usually using `.graphql` and [`VSCode syntax highlighter`](https://marketplace.visualstudio.com/items?itemName=kumar-harsh.graphql-for-vscode) uses `gql` in its examples.

Reply:
I see from https://github.com/facebook/graphql/issues/203 that `.graphql` is used by FB internally so I guess being opinionated is alright there. It will be trivial to add the other extension if needed later

Reply:
Removed in d239f56